cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -162,6 +162,11 @@
   expect(@content_type).to match(content_type)
 end
 
+Then(/^the HTTP response content type is base64 encoded$/) do
```

Comment:
this seems incorrect. you don't verify that the content type is base64 encoded.

Reply:
`  decoded_body = JSON.parse Base64.strict_decode64(@result.body)` This is decoding and json parsing. In my opinion that verifies the encoding. I can add keys validation but again I used the main authentication test as an example. I cant change it in both of them

Reply:
yes but it says `the... content type is base64`

Reply:
i think that it is stronger to not only validate that the returned object is a hash but that it is a token for `alice`. please see authn-azure (or any other) for reference. even if the test above doesn't implement it i think it is best that you do.

Reply:
Ok thanks

Reply:
The audit isn't enough? @orenbm

Reply:
let me ask you this - if you change the line of code that returns the token (`render content_type => authn_token`) to `render content_type => "liav is the kingggg`, will your current test still pass?

Reply:
So both